tests/integration/s2n_dynamic_record_size_test.py

```diff
@@ -284,7 +269,8 @@ def main():
     
     failed = 0
     print("\n\tRunning s2n dynamic record size tests\n\t")
-    failed += test(host, port, test_ciphers, int(file_size / 2))
+    threshold = 10000  # threshold size set independently of file size
```

Comment:
The testing is a lot more stable if we set this bound independently of the file size. 

However, we might wish to add an additional check that the file we are transferring is > 10000B otherwise we wouldn't see high throughput mode put into action.

Reply:
Added small test in [#1e2620f](https://github.com/awslabs/s2n/pull/2292/commits/1e2620fc4a7e9085f5b52b9796bc2799ce3edc93) to verify that file size of `./data/test_buf > 10000` could always change this back to something like `threshold = int(file_size)/4` as this is ~9000B.